net/ampr-ripd/files/ampr-ripd-init

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh /etc/rc.common
+
+START=95
+STOP=10
+
+if [ -f /etc/ampr-ripd.conf ]; then
+   source /etc/ampr-ripd.conf
+else
+   echo /etc/ampr-ripd.conf not found!  Exiting...
+   exit 1
+fi
+
+if [ ! -d /var/lib/ampr-ripd ]; then
+   mkdir -p /var/lib/ampr-ripd
+fi
+
+start() {
+   ip tunnel change ttl 64 mode ipip tunl0
```

Comment:
Where is `tunl0` coming from? And should it be a configuration option?

Reply:
I still feel like `tunl0`, that is used in multiple places, is unclear, and can be named and documented better. What if there's already a `tunl0`?